saltstack/states/github-actions/runner.sls

```diff
@@ -1,6 +1,8 @@
 github:
   user.present:
     - groups:
+      # TODO: remove wheel group
+      # TODO: add docker group
```

Comment:
I don't think the wheel group will be needed by anything, but we will need the docker group in order to do docker builds...  Either way we are giving a privilege escalation path for the host, but I don't think we can avoid it. We could potentially look configured for rootless use, but that is likely to be a pretty deep rabbit hole to chase.